src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -166,7 +167,17 @@ public void commitAddressBook() {
 
         @Override
         public void addEvent(Event toAdd) {
+            throw new AssertionError("This method should not be called.");
+        }
+
```

Comment:
What is the purpose of the addEvent test?

Reply:
This is part of the ModelStub that just implements all the Model interface methods